var/spack/repos/builtin/packages/py-pystan/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPystan(PythonPackage):
+    """PyStan is a Python interface to Stan, a package for Bayesian inference."""
+
+    homepage = "https://mc-stan.org/"
+    pypi     = "pystan/pystan-3.4.0.tar.gz"
+
+    maintainers = ['haralmha']
+
+    version('3.4.0', sha256='325e2fb0ab804555c05a603e0c9152ab11fcc3af01f3e9a9ff9fe9954b93184f')
+
+    depends_on('python@3.8:3', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools', type=('build', 'run'))
```
`pkg_resources` is needed at run-time.